runtime/lua/tohtml.lua

```diff
@@ -481,7 +481,7 @@ local function styletable_treesitter(state)
 end
 
 --- @param state vim.tohtml.state
---- @param extmark {[1]:integer,[2]:integer,[3]:integer,[4]:vim.api.keyset.set_extmark|any}
+--- @param extmark [integer,integer,integer,vim.api.keyset.set_extmark|any]
```

Comment:
While using tuple syntax here as well will be good for consistency, but I am actually not very sure this would read better than the previous one.

Reply:
It's shorter; that alone helps.

Reply:
Let me know if I should revert it.

Reply:
Since we never enforced formatting and we don't have an automatic formatter or any established convention for luadoc annotation, I fully agree with @dundargoc that one should not nitpick formatting/style issues and that it's OK to have inconsistent whitespaces for luadoc annotations, However, my point was that this line is not quite easy to read, so please consider add some whitespaces if you don't mind.

```suggestion
--- @param extmark [integer, integer, integer, vim.api.keyset.set_extmark|any]
```

Reply:
done